### ISSUE TITLE
4/19/2022 Update

### ISSUE THREAD

**Issue by noto777** [alfficcadenti/splinterlands-bot#115]
Todays update broke the bot. It no longer selects teams

**Comment by alfficcadenti** [comment]
can you share some error log?

**Comment by yatumnoo** [comment]
![Capture](https://user-images.githubusercontent.com/93625414/164130519-f0a35a2b-3bd9-4423-aa09-61ef06bacca5.PNG)

**Comment by naveen9036** [comment]
Yeah actually even im facing that issues , after the current `update.

```
getting user quest info from splinterlands API...
getting user cards collection from splinterlands API...
cards retrieved: 119
delegated cards size: 5 [ 28, 90, 131, 167, 440 ]
nani9036  deck size: 119
claim daily quest setting: true Quest details:  { name: 'Stir the Volcano', spli
nter: 'fire', total: 5, completed: 5 }
no quest reward to be claimed waiting for the battle...
check #find_opponent_dialog modal visibility
waiting for selector `#find_opponent_dialog` failed: timeout 10000ms exceeded
waiting for create team button
btn--create-team not detected
Launch battle iter-[1]
waiting for battle button
Battle button clicked
check #find_opponent_dialog modal visibility
find_opponent_dialog visible
waiting for an opponent...
find_opponent_dialog has closed
opponent found?
waiting for create team button
start the match
Error: {
  mana: 'no mana',
  rules: '',
  splinters: [],
  myCards: [
    157, 158, 159, 160, 395, 396, 397, 398, 399, 161, 162, 163,
    167, 400, 401, 402, 403, 440, 168, 169, 170, 171, 381, 382,
    383, 384, 385, 172, 173, 174, 178, 386, 387, 388, 389, 437,
    179, 180, 181, 182, 334, 367, 368, 369, 370, 371, 183, 184,
    185, 189, 372, 373, 374, 375, 439, 146, 147, 148, 149, 409,
    410, 411, 412, 413, 150, 151, 152, 156, 414, 415, 416, 417,
    135, 135, 136, 137, 138, 353, 354, 355, 356, 357, 139, 140,
    141, 145, 358, 359, 360, 361, 438, 224, 190, 191, 192, 157,
    423, 424, 425, 426,
    ... 19 more items
  ],
  preferredCards: [ 28, 90, 131, 167, 440 ]
} []
Error handling browser not opened, internet connection issues, or battle cannot
start: Error: NO TEAMS available to be played
    at startBotPlayMatch (C:\Users\Naveen Kumar\Documents\workspace-spring-tool-
suite-4-4.6.2.RELEASE\splinterlands-bot\index.js:465:19)
    at async run (C:\Users\Naveen Kumar\Documents\workspace-spring-tool-suite-4-
4.6.2.RELEASE\splinterlands-bot\index.js:612:9)
    at async startSingle (C:\Users\Naveen Kumar\Documents\workspace-spring-tool-
suite-4-4.6.2.RELEASE\splinterlands-bot\main.js:83:5)
    at async C:\Users\Naveen Kumar\Documents\workspace-spring-tool-suite-4-4.6.2
.RELEASE\splinterlands-bot\main.js:93:9
Closing battle 4/20/2022, 1:46:02 `PM`` 
```
